arangod/Aql/SubqueryEndExecutionNode.cpp

```diff
@@ -80,8 +77,6 @@ void SubqueryEndNode::toVelocyPackHelper(VPackBuilder& nodes, unsigned flags,
     _inVariable->toVelocyPack(nodes);
   }
 
-  nodes.add("isModificationSubquery", VPackValue(isModificationNode()));
```

Comment:
I can't see any deserialization anywhere :)

Reply:
Ok, I lied, the deserialization is there (and was deleted)

```
58:      _isModificationSubquery(basics::VelocyPackHelper::getBooleanValue(
59:          base, "isModificationSubquery", false)) {}
```

but this *should* not be a problem, since in a rolling upgrade the DB Server gets upgraded first and never tries to read this information again.